### PR TITLE
[PAPP-32297] ssh: Bugfix - Removing the affected paramiko package from the app

### PR DESCRIPTION
**Pull request by bbielinski-splunk**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The manual_readme_content.md has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe):

## Security Considerations (REQUIRED)
- [x] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers),
  please document them in the `manual_readme_content.md`.
- [x] If this is a new connector or you are adding new actions
    - [x] Please document in the `manual_readme_content.md` all methods (eg, OAuth) used to authenticate
      with the service that the connector is integrating with.
    - [x] If any actions are unable to run on SOAR Cloud, please document this in the `manual_readme_content.md`.
- [x] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [x] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [x] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

## Release Notes (REQUIRED)
- Removed the affected paramiko package from the app [PAPP-32297]

## What is the current behavior? (OPTIONAL)
n/a

## What is the new behavior? (OPTIONAL)
n/a

## Other information (OPTIONAL)
n/a

## Pay close attention to (OPTIONAL)
n/a

## Screenshots (if relevant)
n/a

---
Thanks for contributing!

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 beautifulsoup4==4.9.1
 cryptography==39.0.2
```
Comment (tonyc-phantom) [inline requirements.txt:2]: I was looking at the previous commits after the last release, and I actually see that the cryptography package was added to the requirements.txt and then downgraded - is there a reason why you added it? It is my fault for missing it as I approved it. (https://github.com/splunk-soar-connectors/ssh/pull/15/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R2) Just wondering if it is possible to remove this package from requirements.txt or not?

**Review by tonyc-phantom — CHANGES_REQUESTED** [review]

File: phssh.json [hunk 2]
```diff
@@ -2143,44 +2143,5 @@
...
-            },
-            {
-                "module": "soupsieve",
-                "input_file": "wheels/py3/soupsieve-2.3.2.post1-py3-none-any.whl"
-            }
-        ]
-    }
-}
\ No newline at end of file
+    "pip_dependencies": {}
```
Comment (mposluszny-splunk) [inline phssh.json:2146]: do we need to keep empty key here?

**Review by mposluszny-splunk — APPROVED** [review]

**Review by bb-splunk — APPROVED** [review]

File: phssh.json (hunk 2 above)
Comment (bbielinski-splunk) [reply phssh.json]: No, removed, thanks

File: requirements.txt (hunk 1 above)
Comment (bbielinski-splunk) [reply requirements.txt]: UPDATE: Bumped cryptograpgy to 41.0.6 due to new vulnerability detected in previous versions.

**Pull request merged**